src/backends/unicodeplots.jl

```diff
@@ -41,6 +41,7 @@ function unicodeplots_rebuild(plt::Plot{UnicodePlotsBackend})
         end
 
         kw = (
+            compact = true,
```

Comment:
I was a little confused by this line. I did not look closely, so I thought you are adding a new kw "compact" to rotate the labels automatically (but apparenetly not really)

Reply:
It's a merge conflict error, will remove.